src/views/inspector/InspectorViewProvider.ts

```diff
@@ -111,11 +121,30 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
       } else if (data.type === 'jsonReadonly') {
         await executeOpenReadonlyDocumentCommand(data.value)
       }
-      webviewView.webview.html = await this._getHtmlForWebview(webviewView.webview)
+
+      // check if we need to fetch feature configs for the feature currently selected
+      if (this.selectedFolder &&
+          this.selectedType === 'Feature' &&
+          !this.featureConfigsMap[this.selectedKey]
```

Comment:
do we need this extra map/cache? we should already we caching the feature configs by id in the StateManager, so calling `getFeatureConfigurations` should returned the stored values if available

Reply:
I was experiencing that it was calling the feature configuration cli method everytime I switched, I'll look into if the caching is correct

Reply:
@laurawarr I actually think it was mostly so that we can make loading it async, so we don't have to await it when we switch to a feature and so that when we refresh the inspector it will have the feature configs immediately